client/src/pages/ModalBookingScreen.js

```diff
@@ -66,16 +78,31 @@ function ModalBookingScreen({ bookingDate }) {
 							</Form.Field>
 							<Divider inverted />
 							<label>Date: </label>
+<<<<<<< Updated upstream
 							<input
 								type="text"
 								value={formatBookingDate(bookingDate)}
 								disabled
 							/>
+=======
+							<input type="text" value={bookingDate} />
+>>>>>>> Stashed changes
 						</Segment>
 						<Segment>
 							<Form.Field>
-								<Checkbox label="I don't care where I sit" defaultChecked />
+								<Checkbox
```

Comment:
to have a "controlled" Checkbox, you should link it up with the checkbox state, as in https://react.semantic-ui.com/modules/checkbox/#usage-remote-control